keras2pmml/__init__.py

```diff
@@ -16,6 +16,11 @@
 }
 
 
+def _get_activations(model_config):
```

Comment:
This means you are no longer going to support Keras version < 2.0 - is this intentional? You can avoid this by accessing data in a different way for both versions of API while preserving compatibility.